bigchaindb/core.py

```diff
@@ -441,6 +417,19 @@ def get_owned_ids(self, owner):
 
         return owned
 
+    def get_transactions_filtered(self, asset_id=None, operation=None):
+        """
+        Get a list of transactions filtered on some criteria
+        """
+        if not asset_id:
+            raise ValueError("Need asset_id")
```

Comment:
Instead of checking for the `asset_id`, why not making it mandatory? It can still be a keyword argument:

```python
def get_transactions_filtered(self, *, asset_id, operation=None):
    # ...
```